hips/tiles/tests/test_surveys.py

```diff
@@ -0,0 +1,14 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from astropy.utils.data import get_pkg_data_filename
+from ..surveys import HipsSurveyList
+
+
+class TestHipsSurveyList:
+    @classmethod
+    def setup_class(cls):
+        filename = get_pkg_data_filename('data/surveys.txt')
+        cls.hips_survey_list = HipsSurveyList.read(filename)
+
+    def test_surveys(self):
+        len_precomp = 335
+        assert len(self.hips_survey_list.surveys) == len_precomp
```

Comment:
I'd suggest to change the test file to just have a few entries (e.g. 3).
Also, please add an assert or two on some data value in the table, which shows that it was parsed correctly.